models/genmodels.go

```diff
@@ -45,18 +45,56 @@ func Generate(packagePath string, s spec.Swagger) error {
 		return fmt.Errorf("error generating go-swagger models: %s", err)
 	}
 
-	if err := generateOutputs(packagePath, s); err != nil {
+	if err := generateOutputs(basePath, s); err != nil {
 		return fmt.Errorf("error generating outputs: %s", err)
 	}
-	if err := generateInputs(packagePath, s); err != nil {
+	if err := generateInputs(basePath, s); err != nil {
 		return fmt.Errorf("error generating inputs: %s", err)
 	}
+	if err := CreateModFile("models/go.mod", basePath, packageName); err != nil {
+		return fmt.Errorf("error creating go.mod file: %s", err)
+	}
 	return nil
 }
 
-func generateInputs(packagePath string, s spec.Swagger) error {
+//CreateModFile creates a go.mod file for the client module.
+func CreateModFile(path string, basePath string, packageName string) error {
+	absPath := basePath + "/" + path
+	f, err := os.Create(absPath)
+
+	if err != nil {
+		return err
+	}
+
+	defer f.Close()
+	modFileString := `
+module ` + packageName + `/models
+
+
+go 1.16
+
+require (
+	github.com/go-openapi/errors v0.20.2
+	github.com/go-openapi/strfmt v0.21.2
+	github.com/go-openapi/swag v0.21.1
+	github.com/go-openapi/validate v0.22.0
+	github.com/google/go-cmp v0.5.5 // indirect
+	github.com/google/uuid v1.1.2 // indirect
+	golang.org/x/net v0.0.0-20210614182718-04defd469f4e // indirect
+)
+
+	`
+	_, err = f.WriteString(modFileString)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
Still want to make this a go mod init followed by a bunch of go gets. 
Or even just an empty go.mod with the go get calls. Here and in client/gengo.go